doc/adoc/user_guide.adoc

```diff
@@ -62,9 +69,58 @@ To install the distribution migration system run:
 [listing]
 tux > sudo zypper in SLE15-Migration suse-migration-sle15-activation
 
+== Customize the Migration Process
+
+The migration system reads a custom config file from the system to upgrade.
+The contents of that file allows to influence the behavior of the upgrade
+process. Prior to the start of the upgrade process, create the following
+file:
+
+[listing]
+tux > ssh INSTANCE_USER@IP_OF_INSTANCE 'touch /etc/sle-migration-service.yml'
+
+The custom config file allows for the following settings:
+
+Control Zypper Installation Mode::
+If the upgrade process is used on systems that are not SLES registered
+or for which the SUSE Customer Center has no upgrade path, it's required to
+switch off the use of the migration plugin.
+
+[listing]
+use_zypper_migration_plugin: true|false
```

Comment:
I know this is late, I missed this in one of the commits, can we rename the option to be just `use_zypper_migration`?

Reply:
yes we can as it's all still in development. But let's handle it in a followup request please.
Could you create an issue for it, then we don't forget. Thanks